app/views/profiles/show.html.erb

```diff
@@ -13,31 +13,30 @@
         </div>
         <div id='friend-request-button' class='absolute -top-4 -right-2 '>
           <span class='bg-blue-200 py-1 px-2 rounded-full opacity-50'>Request Friend</span>
-          <% if current_user == @profile.user %>
-            <%= link_to 'Edit', edit_user_profile_path(@user), id:'edit-profile-button', class:'bg-blue-200 py-2 px-2 rounded-full w-9', data: {turbo_frame: 'edit-profile-modal'}%>
+          <% if current_user == @user %>
+            <%= link_to 'Edit', edit_profile_path(@profile.user.unique_name), id:'edit-profile-button', class:'bg-blue-200 py-2 px-2 rounded-full w-9', data: {turbo_frame: 'edit-profile-modal'}%>
           <% end %>
         </div>
       </div>
       <div id='about-body' class='relative pt-14'>
-        <div id='name' class='font-bold font-sans'><%= @profile.user.display_name%> - <%=@profile.slug%></div>
-        <div id='about-text' class='mt-4'><%= @profile.description%></div>
+        <div id='name' class='font-bold font-sans'><%= @user.display_name %> (@<%= @user.unique_name %>)</div>
+        <div id='about-text' class='mt-4'><%= @profile.description %></div>
       </div>
     </div>
   </div>
   <div id='content-body' class='marble-background'>
     <ul id='content-nav' class='flex mt-12 px-36 border-b border-yellow-600 divide-x divide-yellow-600'>
       <li class='px-2'>
-        <%= link_to 'Posts', user_profile_path(@profile.user.id), data: { turbo_frame: 'content' } %>
+        <%= link_to 'Posts', profile_path(@user.unique_name), data: { turbo_frame: 'content' } %>
       </li>
       <li class='px-4' > 
-        <%= link_to 'Friends', friends_user_profile_path(@profile.user.id), data: { turbo_frame: 'content' } %>
+        <%= link_to 'Friends', friends_profile_path(@user.unique_name), data: { turbo_frame: 'content' } %>
       </li>
       <li class='px-2 opacity-50'>Media</li>
     </ul>
     <%= turbo_frame_tag 'content', class: 'marble-background' do %>
       <div id='posts'>
-        <%= render partial: 'posts/post', collection: @posts %>
```

Comment:
This was doubling the posts

Reply:
Won't this start the pagination on the second page of posts though?

Reply:
resolved in [d475261](https://github.com/emmahyde/grouper/pull/41/commits/d4752612352c8fca9915d32c34eb8343a76fabf4)